daily-challenges/2024-09/18/huynhtram1504.spec.ts

```
@@ -0,0 +1,50 @@
+import{test, expect} from "@playwright/test";
+/*Viết một hàm JavaScript có tên calculateAge để tính số tuổi của một người dựa trên năm sinh của họ.
+In ra số tuổi tương ứng với năm hiện tại.
+Nếu năm sinh lớn hơn năm hiện tại, in ra một thông báo lỗi "Năm sinh không hợp lệ."*/
+function calculateAge(year: number){
+    let currentTime = new Date();
+    let currentYear = currentTime.getFullYear();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'currentTime' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)